frontend/components/panels/WalletAuthorizationCheckPanel.vue

```diff
@@ -79,7 +80,7 @@ export default Vue.extend({
             }
             if (!this.isWalletAuthorized) {
                 return {
-                    name: 'incorrect',
+                    name: this.disabled ? 'inactive' : 'incorrect',
```

Comment:
Just wondering, how does those changes ended up in the PR for `SurplusAuctionTransaction`?